glasses618/rails101
# [Feature] User console for groups and posts

Feature:
1. Add group and post console.
2. Users can see groups they have joined in the group console.
3. Users can see posts they have posted in the post console.
4. Users can edit and delete posts in the post console.

How to test:

Run rails server `$rails s`, and open http://localhost:3000 with your browser. 
After login, you can enter group and user console via the navigation bar.
In the group console, you can see the groups you have joined.
In the user console, you can see the posts you have posted. What's more, you can also edit and delete those posts.
